### PR TITLE
add error msg for invalid -e option

### PR DESCRIPTION
**Pull request by carolynhu**
https://github.com/rancher/rancher/issues/9132

### CODE REVIEW

File: cmd/run.go [hunk 1]
```diff
@@ -294,8 +295,14 @@ func serviceRun(ctx *cli.Context) error {
 	for _, env := range ctx.StringSlice("env") {
 		parts := strings.SplitN(env, "=", 2)
 		value := ""
+
 		if len(parts) > 1 {
 			value = parts[1]
+
+			if len(parts[0]) == 0 {
```
Comment (joshwget) [inline cmd/run.go]: Can you switch to `parts[0] != ""`? A bit more idiomatic.

File: cmd/run.go [hunk 2]
```diff
@@ -294,8 +295,14 @@ func serviceRun(ctx *cli.Context) error {
...
+				errMsg := fmt.Sprintf("invalid argument \"%s\" for e: invalid environment variable: %s\nSee 'rancher run --help'.", env, env)
```
Comment (joshwget) [inline cmd/run.go]: This is a bit verbose. Maybe we can just use the second half of the error?

Comment (carolynhu) [inline cmd/run.go]: I follow docker's error message on that, if we run: docker run -e "=bar" ubuntu, the msg will be:
invalid argument "=bar" for e: invalid environment variable: =bar
See 'docker run --help'.

File: cmd/run.go (hunk 1 above)
Comment (carolynhu) [inline cmd/run.go]: sure

**Comment by joshwget** [comment]
LGTM

**Pull request merged**